gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -1638,6 +1638,48 @@ static PyObject* GemRB_Control_SetText(PyObject* self, PyObject* args)
 	Py_RETURN_NONE;
 }
 
+PyDoc_STRVAR( GemRB_Window_SetKeyPressEvent__doc,
```

Comment:
I've updated the doc, but it is already rewritten internally. If not needed anymore, then we can also purge it.